sdks/js/packages/core/react/components/organization/tokens/index.tsx

```diff
@@ -0,0 +1,141 @@
+import { Flex, Image, Text } from '@raystack/apsara';
+import { styles } from '../styles';
+import Skeleton from 'react-loading-skeleton';
+import tokenStyles from './token.module.css';
+import { useFrontier } from '~/react/contexts/FrontierContext';
+import { useEffect, useState } from 'react';
+import coin from '~/react/assets/coin.svg';
+
+interface TokenHeaderProps {
+  billingSupportEmail?: string;
+  isLoading?: boolean;
+}
+
+const TokensHeader = ({ billingSupportEmail, isLoading }: TokenHeaderProps) => {
+  return (
+    <Flex direction="column" gap="small">
+      {isLoading ? (
+        <Skeleton containerClassName={tokenStyles.flex1} />
+      ) : (
+        <Text size={6}>Tokens</Text>
+      )}
+      {isLoading ? (
+        <Skeleton containerClassName={tokenStyles.flex1} />
+      ) : (
+        <Text size={4} style={{ color: 'var(--foreground-muted)' }}>
+          Oversee your billing and invoices.
+          {billingSupportEmail ? (
+            <>
+              {' '}
+              For more details, contact{' '}
+              <a
+                href={`mailto:${billingSupportEmail}`}
+                target="_blank"
+                style={{ fontWeight: 400, color: 'var(--foreground-accent)' }}
+              >
+                {billingSupportEmail}
+              </a>
+            </>
+          ) : null}
+        </Text>
+      )}
+    </Flex>
+  );
+};
+
+function getFormatedNumberString(num: Number = 0) {
```

Comment:
Let's move this to a common util. We must have used this earlier in billing as well, right?